docs/en/reference/sql/dql/JOIN_CLAUSE.md

```diff
@@ -18,51 +19,219 @@ JoinType ::= 'LAST'
+| JOIN Clause                | **``✓``**    | **``✓``**           | **``✓``**           | The Join clause indicates that the data source comes from multiple joined tables. OpenMLDB currently only supports LAST JOIN. During Online Serving, please follow [the specification of LAST JOIN under Online Serving](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.md#online-servinglast-join)                                                      |
 
-### LAST JOIN without ORDER BY
 
-#### Example: **LAST JOIN Unsorted Concatenation**
 
-```sql
--- desc: simple spelling query without ORDER BY
 
-SELECT t1.col1 as t1_col1, t2.col1 as t2_col2 from t1 LAST JOIN t2 ON t1.col1 = t2.col1
-```
+### LAST JOIN without ORDER BY
 
-When `LAST JOIN` is spliced ​​without sorting, the first hit data row is spliced
+#### Example1: Unsorted LAST JOIN
+
+The unsorted `LAST JOIN` will concat every row of the left table with the last matching row of the right table.  
 
 ![Figure 7: last join without order](../dql/images/last_join_without_order.png)
 
 
-Take the second row of the left table as an example, the right table that meets the conditions is unordered, there are 2 hit conditions, select the last one `5, b, 2020-05-20 10:11:12`
+Take the second row of the left table as an example. The right table is unordered, and there are 2 matching rows. The last one `5, b, 2020-05-20 10:11:12` will be joined with the second row of the left.  
 
 ![Figure 8: last join without order result](../dql/images/last_join_without_order2.png)
 
 The final result is shown in the figure above.
 
+##### SQL Example using OpenMLDB CLI
+The following SQL commands created the left table t1 as mentioned above and inserted corresponding data.
+In order to check the results conveniently, it is recommended to create index on `col1` and use `std_ts` as timestamp. It doesn't matter if you create t1 without index, since it doesn't influence the concatenation in this case.
+```sql
+>CREATE TABLE t1 (id INT, col1 STRING,std_ts TIMESTAMP,INDEX(KEY=col1,ts=std_ts));
+SUCCEED
+>INSERT INTO t1 values(1,'a',20200520101112);
+SUCCEED
+>INSERT INTO t1 values(2,'b',20200520101114);
+SUCCEED
+>INSERT INTO t1 values(3,'c',20200520101116);
+SUCCEED
+>SELECT * from t1;
+ ---- ------ ----------------
+  id   col1   std_ts
+ ---- ------ ----------------
+  1    a      20200520101112
+  2    b      20200520101114
+  3    c      20200520101116
+ ---- ------ ----------------
+
+3 rows in set
+```      
+The following SQL commands created the right table t2 as mentioned above and inserted corresponding data.
+
+```{note}
+The storage order of data rows is not necessarily the same as their insert order. And the storage order will influence the matching order when JOIN.
```

Comment:
The physical storage order of rows is not necessarily the same as their insertion order. However, the storage order will affect the matching order when performing the LAST JOIN.